### DISCUSSION TITLE
ddev on AWS EC2 instance

### DISCUSSION THREAD

**Discussion by martingrondein-nls** [ddev/ddev#3917] [General]
Has anyone got a guide or had any luck in getting ddev running on an AWS EC2 instance (preferably micro with RDS) ?

**Comment by rfay** [comment]
I've done it many times; it's easy. Just follow the linux instructions. 

* Install docker: https://ddev.readthedocs.io/en/latest/users/docker_installation/#linux-installation-docker
* Install ddev: https://ddev.readthedocs.io/en/latest/#linux-macos-and-windows-wsl2-install-script

If you want it to be accessible from elsewhere, you have to do other things. See https://ddev.readthedocs.io/en/latest/users/alternate-uses/#alternate-uses-for-ddev-local and take a look at the casual webhosting section in particular.
